### DISCUSSION TITLE
Archer T4U Plus V1 working

### DISCUSSION THREAD

**Discussion by Filipe** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#327] [General]
The TP-Link Archer T4U Plus V1 is working with the Wireless USB Big Sur Adapter V17 (with SIP enabled).

I'm using a 2018 MacBook Pro with macOS Sonoma 14.1.1.

Thank you very much! 😊

**Comment by chris1111** [comment]
Thanks for sharing , Glad it work great for you 😀
Enjoy!
